ui/v2.5/src/index.scss

```diff
@@ -61,6 +61,9 @@ body {
 }
 
 .main {
+  padding-left: env(safe-area-inset-left);
+  padding-right: env(safe-area-inset-right);
```

Comment:
This breaks styling on other viewports, since the `safe-area-inset-x` values are 0, when the bootstrap `container-fluid` class uses `15px` as the left and right padding values. It also appeared to provide too much padding on my Pixel device when in landscape mode and added a horizontal scrollbar. If I removed these lines (leaving the `viewport-fit` change), then the page rendered correctly.

I tried the following which worked on my desktop PC, and it eliminated the scrolling on the Pixel, but still rendered with too much left padding in landscape mode:
```
padding-left: calc(max(env(safe-area-inset-left), 15px));
padding-right: calc(max(env(safe-area-inset-right), 15px));
```